utils/forest_utils/src/io/mod.rs

```diff
@@ -29,6 +29,11 @@ pub fn set_user_perm(file: &File) -> Result<()> {
     Ok(())
 }
 
+#[cfg(not(unix))]
+pub fn set_user_perm(file: &File) -> Result<()> {
```

Comment:
Add some comments? e.g. this is backfill function etc.

Reply:
Since we don't have CI on windows, maybe structure it in a more reliable way?

e.g. 
```rust
pub fn set_user_perm(file: &File) -> Result<()> {
    #[cfg(unix)]
    set_user_perm_unix(file)?;
    Ok(())
}

#[cfg(unix)]
pub fn set_user_perm_unix(file: &File) -> Result<()> {
    ...
}
```